index.html

```diff
@@ -129,9 +129,17 @@
     </script>
     <p class="choice-title">Do you know:</p>
     <script>
-        addCategories([["py", "Python"], ["java", "Java"], ["sh", "Shell/Makefile/Autoconf"],
-                       ["js", "JavaScript"], ["cpp", "C/C++"], ["html", "HTML/CSS"], ["xml", "XML/XUL"],
-                       ["perl", "Perl"]]);
+        addCategories([
+          ["cpp", "C/C++"],
+          ["html", "HTML/CSS"],
+          ["java", "Java"],
+          ["js", "JavaScript"],
+          ["perl", "Perl"],
+          ["py", "Python"],
+          ["rs", "Rust"],
```

Comment:
Don't you need to add something to the list at https://github.com/whimboo/bugsahoy/blob/ea3e94b0957eafb4cd0917bf920c91db295f2280/magic.js#L348 as well? Does this return any results when you test it locally right now?

Reply:
Oh, you are right. I searched the code for `py` to find other places which needs to be updated but didn't find any. Strange that there is. Adding such a line clearly makes the bug appear.